betty/project/extension/__init__.py

```diff
@@ -58,12 +60,21 @@ def __init__(self, project: Project):
     async def new_for_project(cls, project: Project) -> Self:
         return cls(project)
 
+    # @todo Deprecate this.
     def register_event_handlers(self, registry: EventHandlerRegistry) -> None:
         """
         Register event handlers with the project.
         """
         pass
 
+    async def dispatch_jobs(
```

Comment:
Implementations are still quite verbose. More verbose than the event handler registration this will eventually replace. Can we make this as simple as that?